chutney/server/src/main/java/com/chutneytesting/execution/domain/PurgeServiceImpl.java

```diff
@@ -228,11 +229,13 @@ private PurgeExecutionService(
          */
         Set<Long> purgeExecutions() {
             Set<Long> deletedExecutionsIds = new HashSet<>();
+            LocalDateTime twentyFourHoursEarlier = LocalDateTime.now().minusHours(24);
```

Comment:
Add a constructor parameter for the number of hours excluding executions (maybe even have two, one for scenario and one for campaign).
All the exisiting tests could now use this constructor with 0 as the value for this future configuration value.